code/game/objects/items/implants/implantswap.dm

```diff
@@ -0,0 +1,65 @@
+/datum/swap
```

Comment:
This entire datum isn't needed. It's only used to hold two variables and a proc. Either turn it into a full scale component or remove it.